packages/frontend/src/features/sqlRunner/components/Header/HeaderView.tsx

```diff
@@ -31,6 +33,8 @@ export const HeaderView: FC = () => {
     const savedSqlChart = useAppSelector(
         (state) => state.sqlRunner.savedSqlChart,
     );
+    const [isAddToDashboardModalOpen, addToDashboardModalHandlers] =
+        useDisclosure();
```

Comment:
could you use a selector and store this modal opening state in the sqlRunner slice?
Similar to `deleteChartModal.isOpen` below 👍